Cargo.toml

```diff
@@ -24,12 +24,12 @@ borsh = "0.9"
 serde = "1.0.127"
 serde_json = "1.0.66"
 
-near-primitives = "0.11.0"
-near-chain-configs = "0.11.0"
-near-jsonrpc-primitives = "0.11.0"
+near-primitives = "0.12.0"
+near-chain-configs = "0.12.0"
+near-jsonrpc-primitives = "0.12.0"
 
 [dev-dependencies]
-near-crypto = "0.11.0"
+near-crypto = "0.12.0"
```

Comment:
See https://github.com/near/near-jsonrpc-client-rs/pull/44

Reply:
@itegulov, on second thought, seeing as I still need to merge https://github.com/near/near-jsonrpc-client-rs/pull/38 that depends on nearcore version bump before client version bump, can you reduce the scope of this PR to just bumping nearcore versions?

Reply:
yeah, sure

Reply:
Done